organization/general_rules_and_guidelines.rst

```diff
@@ -0,0 +1,32 @@
+General rules and guidelines
+============================
+
+This page contains guidelines relevant for contributing to any area of Godot.
+
+When contributing to any particular area, please refer to its respective guidelines additionally.
+
+Code of conduct
+---------------
+
+When you contribute to Godot, we expect that you respect our `Code of Conduct <https://godotengine.org/code-of-conduct/>`__.
+
+Contributing with AI and large language models
+----------------------------------------------
+
+The use of AI to contribute to Godot is discouraged.
+We acknowledge that AI can be useful, but we are convinced that effort by humans results in better and more relevant
+contributions.
```

Comment:
Since we want to use "prohibited" later I guess we can upgrade this statement a notch.
```suggestion
The use of AI to contribute to Godot is **highly discouraged**.
We acknowledge that AI can be useful, but we are convinced that human effort results in better and more relevant
contributions.
```

Reply:
Then you may just say discouraged but still in bold

Reply:
What's the motivation behind making it bold?

Reply:
Mmmm... ease of reading, I suppose. If someone wants the juice of this paragraph, it's highlighted in bold right there.